site/src/modules/workspaces/DynamicParameter/DynamicParameter.tsx

```diff
@@ -175,7 +176,12 @@ const ParameterField: FC<ParameterFieldProps> = ({
 					disabled={disabled}
 				>
 					<SelectTrigger>
-						<SelectValue placeholder="Select option" />
+						<SelectValue
+							placeholder={
+								(parameter.styling as { placeholder?: string })?.placeholder ||
```

Comment:
I see that we have this in the previous code but is there no way we can do this without type casting?